### DISCUSSION TITLE
SSO Authentication

### DISCUSSION THREAD

**Discussion by jonathanviber** [Kong/insomnia#4726] [Ideas] [answered]
There are 2 features that would enable companies (including mine) to use the paid version. Either one would work

**OKTA Authentication**
Allow login to the web portal via OKTA

**Restricted Google Authentication**
Allow users to authenticate using a gmail account while not allowing them to create a fixed username. If a user leaves a company, the admin can remove them from Insomnia via their gmail account. Currently, if you leave a company, you can continue to log in via your fixed username\password.

**Comment by subnetmarco** [comment, marked as answer]
This has been implemented since September 2023, we now support OIDC and SAML compatible SSO, alongside with more enterprise capabilities. You can [reach out](https://insomnia.rest/pricing/contact) to the sales team to see a full demo.
